about.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>About | Nyckolle G. T. Lucuab</title>
+        <link rel="icon" type="image/png" href="Logos/Favicons/Favicon(White)-32x32.png" sizes="32x32" />
+        <link href="about.css" rel="stylesheet" type="text/css">
+    </head>
+    <body>
+        <div class="header">
+            <div id="navbar">
+                <a class="nav-link" href="index.html">HOME</a>
+                <a class="nav-link" href="about.html">ABOUT</a>
+                <a class="nav-link" href="contact.html">CONTACT</a>
+            </div>
+        </div>
```

Comment:
let's use semantic tags here instead:

```suggestion
        <header class="header">
            <nav id="navbar">
                <a class="nav-link" href="index.html">HOME</a>
                <a class="nav-link" href="about.html">ABOUT</a>
                <a class="nav-link" href="contact.html">CONTACT</a>
            </nav>
        </header>
```

we can also consider using an unordered list for the nav links! That's a common pattern you'll see